chcemvediet/apps/inforequests/tests/test_cron.py

```diff
@@ -197,11 +197,10 @@ def test_inforequest_is_skipped_if_exception_raised_while_checking_it(self):
                 message_set = self._call_cron_job()
         self.assertEqual(message_set.count(), 2)
         self.assertEqual(len(logger.mock_calls), 3)
-        self.assertRegexpMatches(logger.mock_calls[0][1][0], u'Checking if undecided email reminder should be sent failed: <Inforequest: %s>' % inforequests[1].pk)
-        self.assertRegexpMatches(logger.mock_calls[1][1][0], u'Sent undecided email reminder: <Inforequest: %s>' % inforequests[0].pk)
-        self.assertRegexpMatches(logger.mock_calls[2][1][0], u'Sent undecided email reminder: <Inforequest: %s>' % inforequests[2].pk)
+        self.assertRegexpMatches(logger.mock_calls[0][1][0], u'Checking if undecided email reminder should be sent failed: {}'.format(re.escape(repr(inforequests[1]))))
+        self.assertRegexpMatches(logger.mock_calls[1][1][0], u'Sent undecided email reminder: {}'.format(re.escape(repr(inforequests[0]))))
+        self.assertRegexpMatches(logger.mock_calls[2][1][0], u'Sent undecided email reminder: {}'.format(re.escape(repr(inforequests[2]))))
```

Comment:
Tu este pozeram, ze preco vlastne mame `assertRegexpMatches`, ked nikde nevyuzivame vlastnosti regexpu. Nestacilo by assertnut, ze `logger.mock_calls[1][1][0]` obsahuje dany string ako jednoduchy substring?

`assertRegexpMatches` by som pouzival iba ked to naozaj porovnavame vzhladom na regexp. Teda ze v ramci matchu mame nieco ako `.*` a podobne.

Reply:
:ok: